### PR TITLE
jinja2-cli: init at 0.8.2

### PR DESCRIPTION
**Pull request by veprbl**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by superherointj** [comment]
Would make sense to locate this package as a Python package? Then reference it at top-level?

**Comment by veprbl** [comment]
It's really an application.

**Review by superherointj — APPROVED** [review]

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -15620,6 +15620,8 @@ with pkgs;
     jdk = jdk8; # TODO: remove override https://github.com/NixOS/nixpkgs/pull/89731
   };
 
+  jinja2-cli = callPackage ../development/tools/jinja2-cli { };
```
Comment (ckiee) [inline pkgs/top-level/all-packages.nix:15623]:
```suggestion
  jinja2-cli = callPackage ../development/tools/jinja2-cli {
    inherit (python3Packages) python3Packages fetchPypi jinja2 pyyaml toml xmltodict;
  };
```
(Also change the parts that use `python3Packages`)

**Review by ckiee — CHANGES_REQUESTED** [review]

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (thiagokokada) [reply pkgs/top-level/all-packages.nix:15623]: This isn't really problematic for Python applications in the top level though.

File: pkgs/development/tools/jinja2-cli/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{ fetchFromGitHub, lib, python3Packages }:
```
Comment (thiagokokada) [inline pkgs/development/tools/jinja2-cli/default.nix:1]:
```suggestion
{ fetchFromGitHub, lib, python3 }:
```

File: pkgs/development/tools/jinja2-cli/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+python3Packages.buildPythonApplication rec {
```
Comment (thiagokokada) [inline pkgs/development/tools/jinja2-cli/default.nix:3]:
```suggestion
python3.pkgs.buildPythonApplication rec {
```

File: pkgs/development/tools/jinja2-cli/default.nix [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  pname = "jinja2-cli";
+  version = "0.8.2";
+  format = "setuptools";
+
+  src = python3Packages.fetchPypi {
```
Comment (thiagokokada) [inline pkgs/development/tools/jinja2-cli/default.nix:8]:
```suggestion
  src = python3.pkgs.fetchPypi {
```

File: pkgs/development/tools/jinja2-cli/default.nix [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    inherit pname version;
+    sha256 = "sha256-oWuxRUEREo4gb1aMlZOM3vW1oTmSk3j3K7jPYXnhjlA=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
```
Comment (thiagokokada) [inline pkgs/development/tools/jinja2-cli/default.nix:13]:
```suggestion
  propagatedBuildInputs = with python3.pkgs; [
```

File: pkgs/development/tools/jinja2-cli/default.nix [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    jinja2
+    pyyaml
+    toml
+    xmltodict
+  ];
+
+  doCheck = false; # no tests on PyPi
```
Comment (SuperSandro2000) [inline pkgs/development/tools/jinja2-cli/default.nix:20]: but in git https://github.com/mattrobenolt/jinja2-cli/tree/main/tests

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**